cdhweb/static_src/global/elements/page.scss

```diff
@@ -1,3 +1,7 @@
+@view-transition {
+  navigation: auto;
+}
```

Comment:
Progressive enhancement for nice fading page transitions, coming soon to a browser near you.